package.json

```
@@ -11,10 +11,12 @@
     "setup": "yarn install && yarn setup:frontend && yarn setup:backend",
     "setup:frontend": "cd frontend && yarn",
     "setup:backend": "cd backend && yarn",
-    "dev": "yarn clear && concurrently \"cd frontend && yarn dev\" \"cd backend && yarn develop\"",
-    "audit:all": "yarn audit && concurrently \"cd frontend && yarn audit\" \"cd backend && yarn audit\""
+    "dev": "concurrently \"yarn clear\" \"cd frontend && yarn dev\" \"cd backend && yarn develop\"",
+    "audit:all": "concurrently \"yarn audit\" \"cd frontend && yarn audit\" \"cd backend && yarn audit\"",
+    "deadcode": "scripts/check_dead_code.sh",
+    "deadcode:update": "yarn --silent deadcode > scripts/dead_code_snapshot.txt"
```

```suggestion
    "test:deadcode": "scripts/check_dead_code.sh",
    "test:deadcode:update": "yarn --silent deadcode > scripts/dead_code_snapshot.txt"
```
Je trouve ça plus clair (y a un verbe, ce qui est mieux je trouve pour une commande), et ça match le nom des commandes dans les repos